feature-libs/user/account/components/verification-token-form/verification-token-form.component.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  HostBinding,
+  OnInit,
+  ViewChild,
+} from '@angular/core';
+import { UntypedFormGroup } from '@angular/forms';
+import { LAUNCH_CALLER, LaunchDialogService } from '@spartacus/storefront';
+import { Observable } from 'rxjs';
+
+import { VerificationToken } from '@spartacus/user/account/root';
+import { ONE_TIME_PASSWORD_LOGIN_PURPOSE } from '../user-account-constants';
+import { VerificationTokenFormComponentService } from './verification-token-form-component.service';
+
+@Component({
+  selector: 'cx-verification-token-form',
+  templateUrl: './verification-token-form.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class VerificationTokenFormComponent implements OnInit {
+  constructor(
+    protected service: VerificationTokenFormComponentService,
+    protected launchDialogService: LaunchDialogService,
+    private cdr: ChangeDetectorRef
```

Comment:
Any reason why we need `cdr` to be a `private` instead of the `protected`?

Reply:
fixed